services/ui-src/src/components/pages/Dashboard/DashboardPage.test.tsx

```diff
@@ -290,10 +288,8 @@ describe("<DashboardPage />", () => {
     });
 
     test("Cannot unarchive a WP", async () => {
-      const archiveButton = screen.getAllByRole("button", {
-        name: "Archived",
-      })[0];
-      expect(archiveButton).toBeDisabled();
+      const archivedText = screen.getAllByText("Archived")[1];
```

Comment:
Since the mobile and desktop tests are the same, I'm curious what would make the mobile view so different that this additional test is needed? (Just wondering for context, not suggesting any changes.)

Reply:
I assume it's just for test coverage %